test/compilable/cimport.c

```diff
@@ -0,0 +1,10 @@
+// https://issues.dlang.org/show_bug.cgi?id=22666
+
+__import core.stdc.stdarg;
+__import imports.impcimport;
+
+int foo()
+{
+    va_list x;
```

Comment:
Whatabout a non-trivial D symbol like `va_arg`?

Reply:
It's just another symbol.

Reply:
Er, no it's not. va_arg is a template. The way to deal with that is to import another D module that instantiates the va_arg template and presents it to ImportC as a global declaration of a function.